api/rest.go

```diff
@@ -2056,6 +2094,10 @@ func handleUpdateContentUnit(cp utils.ContextProvider, exec boil.Executor, cu *P
 		return nil, NewForbiddenError()
 	}
 
+	if unit.TypeID == common.CONTENT_TYPE_REGISTRY.ByName[common.CT_SOURCE].ID {
+		return nil, NewForbiddenError()
```

Comment:
Semantics. It's more 400 than 403. I'd rather have it bad request instead of forbidden. Also a nice message will be nice for the user.